include/libpldm/oem/ibm/state_set.h

```diff
@@ -17,6 +17,7 @@ enum ibm_oem_pldm_state_set_ids {
 	PLDM_OEM_IBM_PCIE_SLOT_SENSOR_STATE = 32774,
 	PLDM_OEM_IBM_SBE_SEMANTIC_ID = 32775,
 	PLDM_OEM_IBM_SBE_HRESET_STATE = 32776,
+	PLDM_OEM_IBM_BOOT_SIDE_RENAME = 32773,
```

Comment:
I see 2 entries with same state set id 32773 ? PLDM_OEM_IBM_PCIE_SLOT_EFFECTER_STATE and PLDM_OEM_IBM_BOOT_SIDE_RENAME . Can we have any one of them changed?

Reply:
oh wow , very good catch @Pavithrab7 , this would have broken the entire CM/hotplug.